packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -360,6 +373,22 @@ export class FetchInstrumentation extends InstrumentationBase<
     };
   }
 
+  private _applyAttributesAfterFetch(
+    span: api.Span,
+    request: Request | RequestInit,
+    result: Response | FetchError
+  ) {
+    const applyCustomAttributesOnSpan = this._getConfig()
+      .applyCustomAttributesOnSpan;
+    if (applyCustomAttributesOnSpan) {
+      safeExecuteInTheMiddle(
+        () => applyCustomAttributesOnSpan(span, request, result),
+        () => {},
```

Comment:
```suggestion
        (e: Error) => {
          diag.error('applyCustomAttributesOnSpan', e);
        },
```
so that user will see that something went wrong, please check it as I wrote it by hand

Reply:
Nice, added with a small change to check for the presence of `error`